modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -43,48 +45,166 @@ The properties for broker-wide throughput quota balancing are configured at the
 
 == Client throughput limits
 
-Similar to <<broker-wide-throughput-limits>> but for clients, Redpanda provides configurable throughput quotas that apply to an individual client or a group of clients.
+Redpanda provides configurable throughput quotas that apply to an individual client or a group of clients. You can apply a quota for an individual client based on an exact match with its `client_id`, or a group of clients based on IDs that start with a given prefix. 
+
+As of v24.2, client throughput quotas are compatible with the https://cwiki.apache.org/confluence/display/KAFKA/KIP-546%3A+Add+Client+Quota+APIs+to+the+Admin+Client[Kafka API^] (AlterClientQuotas and DescribeClientQuotas), and are separate from quotas configured through cluster configuration in earlier Redpanda versions. The client throughput quotas no longer apply on a per-shard basis, and now limit the rates across a Redpanda broker's node. The quotas are neither shared nor balanced between brokers.
+
+Redpanda supports the following Kafka API-based quota types on clients:
+
+|===
+| Quota type | Description
+
+| producer_byte_rate
+| Limit throughput of produce requests
+
+| consumer_byte_rate
+| Limit throughput of fetch requests
+
+| controller_mutation_rate
+| Limit rate of topic mutation requests, including create, add, and delete partition, in number of partitions per second
+
+|===
+
+You can also apply a default quota for all other client requests that don't have a specific quota configured based on the exact or prefix match on `client_id`. 
```

Comment:
Not sure how to reword because I'm can figure out what your intent is here.
Perhaps:
You can also apply a default quota on all other client requests that don't have a specific quota based on an exact match or `client_id` prefix match.